src/utilities/PlaneAveraging.H

```diff
@@ -20,7 +20,9 @@ public:
     ~PlaneAveraging(){};
 
     void operator()(const amrex::Vector<amrex::Geometry>& Geom,
+               amrex::Vector<amrex::MultiFab*> const& density,
                amrex::Vector<amrex::MultiFab*> const& velocity,
+               amrex::Vector<amrex::MultiFab*> const& mueff,
```

Comment:
Is it time for `PlaneAveraging` to become aware of `FieldRepo`?

Reply:
I think so!